class/defaults.yml

```diff
@@ -1,3 +1,57 @@
 parameters:
   keycloak:
     namespace: syn-keycloak
+    # Hostname should be overwritten on the cluster level
+    hostname: keycloak.example.com
+    # Keycloak Admin
+    admin:
+      secretname: keycloak-admin-user
+      username: admin
+      password: '?{vaultkv:${customer:name}/${cluster:name}/keycloak/admin-password}'
+    # Ingress or Route should be enabled on the distribution level
+    ingress:
+      enabled: false
+    route:
+      enabled: false
+    # Labels can be extended in the config hierarchy by providing further
+    # entries in key `labels`.
+    labels:
+      app.kubernetes.io/name: keycloak
```

Comment:
These labels are already set by the chart, no need to set them here again.
https://github.com/codecentric/helm-charts/blob/master/charts/keycloak/templates/_helpers.tpl#L35

Reply:
since we use `parameters.keycloak.labels` as labelset for the secrets managed by Commodore (cf. main.jsonnet) as well as for the Helm chart, it may make sense to manage the whole labelset here.